_includes/location/offshore-area-federal-production-table.html

```diff
@@ -12,7 +13,7 @@
        class='county-table'>
   <thead>
     <tr>
-      <th>{{ region_name }}</th>
+      <th>{{ locality_name }}</th>
```

Comment:
Makes the column header display as `Region` instead of the particular region's name, in accordance with how we are approaching this on other pages.